packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -1100,7 +1100,7 @@ const AccountSwitcher = props => {
                                                     closeAccountsDialog();
                                                     props.setShouldShowCooldownModal(true);
                                                 } else {
-                                                    props.openRealAccountSignup(account);
+                                                    props.openRealAccountSignup('manage');
```

Comment:
This should be `account`. The function is for adding a new account. I believe manage is for when the user already has a fiat/crypto account.

Reply:
> This should be `account`. The function is for adding a new account. I believe manage is for when the user already has a fiat/crypto account.

I had doubts about this too. In this case I'll ask card owner what she wants to see by button click. Thank you